scripts/sort_contibutors/main.py

```diff
@@ -0,0 +1,69 @@
+import requests
+import json
+import os
+from dotenv import load_dotenv
+
+# It's just for debugging purpose
+def print_list(user_list : list, label : str):
+	print(label)
+	for element in user_list:
+		print(element['username'],element['ranking'])
```

Comment:
instead of printing it like this, i need 2 lists:
1 with html:
```html
<li><a href="https://www.github.com/element['username']"><element['Real name'] @element['username']</a><li>
```

1 consisting of markdown prints:
```md
[element['real name'] @element['username'](https://www.github.com/element['username'])
```